pasta_eln/GUI/projectTreeView.py

```diff
@@ -110,7 +110,9 @@ def execute(self, command:list[Any]) -> None:
             if (oldPath.parent/newFileName).exists():  #ensure target does not exist
               endText = ' was marked for deletion. Save it or its content now to some place on harddisk. It will be deleted now!!!'
               showMessage(self, 'Warning', f'Warning! \nThe folder {oldPath.parent/newFileName}{endText}')
-              if (oldPath.parent/newFileName).exists():
+              if (oldPath.parent/newFileName).is_file():
```

Comment:
What is this 'if' and 'elif' for? I guess, you want to distinguish between file links and real files, but the is_file() is invoked both ways!! How will it work then?

Reply:
Very good catch. Bug: meant something else. See update